dashboard/upload_dashboard.sh

```diff
@@ -17,8 +17,8 @@ ssh -i "$KEY_PATH" $EC2_USER@$EC2_HOST << EOF
     fi
     
     # Create a Python virtual environment and install dependencies
-    python3 -m venv venv
-    source venv/bin/activate
+    python3 -m venv .venv
```

Comment:
Why `.venv` over `venv`?

Reply:
Got errors from EC2 when it was `venv` for some reason beyond my understanding.

Reply:
Using a `.` beforehand suggests that the file is a hidden file, which a `.venv` technically should be